# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/vlc/hotfix-assert.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/include/vlc_aout.h
+++ b/include/vlc_aout.h
@@ -24,6 +24,10 @@
 
 #include <assert.h>
 
+#if !defined(__cplusplus) && !defined(static_assert)
+# define static_assert _Static_assert
+#endif
+
 /**
  * \defgroup audio_output Audio output
  * \ingroup output
